manager/state/raft/testutils/testutils.go

```diff
@@ -295,13 +295,14 @@ func NewNode(t *testing.T, clockSource *fakeclock.FakeClock, tc *cautils.TestCA,
 
 	keyRotator := NewSimpleKeyRotator(raft.EncryptionKeys{CurrentDEK: []byte("current")})
 	newNodeOpts := raft.NodeOptions{
-		ID:             securityConfig.ClientTLSCreds.NodeID(),
-		Addr:           l.Addr().String(),
-		Config:         cfg,
-		StateDir:       stateDir,
-		ClockSource:    clockSource,
-		TLSCredentials: securityConfig.ClientTLSCreds,
-		KeyRotator:     keyRotator,
+		ID:               securityConfig.ClientTLSCreds.NodeID(),
+		Addr:             l.Addr().String(),
+		Config:           cfg,
+		StateDir:         stateDir,
+		ClockSource:      clockSource,
+		TLSCredentials:   securityConfig.ClientTLSCreds,
+		KeyRotator:       keyRotator,
+		DisableStackDump: true,
```

Comment:
Do we want to disable all stack dumps for tests?  It might be useful to have a stack dump for tests that aren't meant to wedge, and just disable it for the single new test that is meant to always wedge (`TestRaftWedgedManager`)